AirCasting/SDSync/MobileSessions/SDCardMobileSessionsSavingService.swift

```diff
@@ -95,7 +95,8 @@ class SDCardMobileSessionsSavingService: SDCardMobileSessionssSaver {
         Log.info("Processing mobile session: \(self.sessionUUID)")
         
         guard let processedSession = getSessionData(sessionUUID: sessionUUID, deviceID: deviceID) else {
-            Log.info("Ignoring session \(self.sessionUUID). Moving forward.")
+
```

Comment:
empty line, done with intention?

Reply:
I am a little spoiled with automatic code formatting tools, maybe I'll add some to our git pipeline if I have a spare minute.